steward/src/config.rs

```diff
@@ -267,6 +268,7 @@ impl Default for SimulateSection {
             tenderly_access_key: "".to_string(),
             tenderly_project_name: "".to_string(),
             tenderly_username: "".to_string(),
+            gravity_address: "0x69592e6f9d21989a043646fe8225da2600e5a0f7".to_string(),
```

Comment:
If we're going to include a hardcoded default here, we probably should have left `GRAVITY_ADDRESS` defined and used it here rather than a magic string.